core/src/main/scala/frameless/CatalystCollection.scala

```diff
@@ -0,0 +1,10 @@
+package frameless
+
+trait CatalystCollection[C[_]]
+
+object CatalystCollection {
+  implicit object arrayObject extends CatalystCollection[Array]
```

Comment:
Could you follow the same pattern than the other type functions?

https://github.com/typelevel/frameless/blob/db2ef8a6bfe194798e6db6ebecd65fd18547bb7b/core/src/main/scala/frameless/CatalystBitwise.scala#L13-L14

Reply:
corrected.